crates/bevy_internal/Cargo.toml

```diff
@@ -187,7 +187,8 @@ bevy_ci_testing = ["bevy_dev_tools/bevy_ci_testing", "bevy_render?/ci_limits"]
 # Enable animation support, and glTF animation loading
 animation = ["bevy_animation", "bevy_gltf?/bevy_animation"]
 
-bevy_sprite = ["dep:bevy_sprite", "bevy_gizmos?/bevy_sprite", "bevy_image"]
+bevy_sprite = ["dep:bevy_sprite", "bevy_image"]
+bevy_sprite_render = ["bevy_sprite", "dep:bevy_sprite_render", "bevy_gizmos?/bevy_sprite"]
 bevy_pbr = [
   "dep:bevy_pbr",
   "bevy_gizmos?/bevy_pbr",
```

Comment:
unrelated to this PR, but just noticed that some of the features below here are wrong, for example bevy_gltf just depends on bevy_image. that should say bevy_pbr. Similarly for gizmos, bevy_image is not enough, same with core pipeline and anti_aliasing